hammer_cli.gemspec

```diff
@@ -27,7 +27,7 @@ EOF
   s.require_paths = ["lib"]
   s.executables = ['hammer', 'hammer-complete']
 
-  s.add_dependency 'clamp', '>= 1.1', '< 1.2.0'
+  s.add_dependency 'clamp', '>= 1.1', '< 2.0.0'
```

Comment:
You use the `line` method introduced in clamp 1.3.1 so that should be the lower bound. We'll need to deal with that in packaging, but I think that's ok.